src/ssh/ssh_process.cpp

```diff
@@ -14,7 +14,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-
+#include <source_location>
```

Comment:
Not a thorough review, but `<source_location>` is a C++20 feature, where we're still on C++17. We're planning to switch to C++20 though: https://github.com/canonical/multipass/pull/4196